src/js/view/view.js

```diff
@@ -303,6 +302,10 @@ function View(_api, _model) {
             }
         });
 
+        if (_floatOnScroll) {
+            _model.on('change:intersectionRatio', _updateFloating);
```

Comment:
It's a bit odd to create a listener for this when you also set it in the same script. In general we use listeners for changes that happen somewhere else in the application, That is why `_setFloatingIntersection` was called directly before these changes.

Reply:
You're saying to call `_updateFloating` directly inside `this.setIntersection`?

Reply:
If that is the only place `model` "intersectionRatio" is set, then yes. Using event listeners to execute functions in the same context is inefficient.

Reply:
Yep, it is.